### DISCUSSION TITLE
Support links and embeds of blocks

### DISCUSSION THREAD

**Discussion by zoni** [zoni/obsidian-export#60] [Feature requests]
Obsidian's [block reference](https://publish.obsidian.md/help/How+to/Link+to+blocks) syntax is currently unsupported. This is a tracking issue for support of this functionality.

**Comment by aadimator** [comment]
Hi, great work on this tool. I've been using a modified version of this for my site generation for months. Thanks

Any updates on this block embedding feature? or any plan?
